api/src/test/java/org/pmiops/workbench/utils/TemporalAssertions.java

```diff
@@ -0,0 +1,32 @@
+package org.pmiops.workbench.utils;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import java.time.Instant;
+import java.time.OffsetDateTime;
+
+public class TemporalAssertions {
+
+  private static final double TIME_TOLERANCE_MILLIS = 100.0;
+
+  public static void assertTimeWithinTolerance(OffsetDateTime actual, OffsetDateTime expected) {
```

Comment:
from our offline conversation: please add a version of this with a tolerance, and a basic test for all methods here which don't yet have one